conventional-changelog-lighthouse/index.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const readFileSync = require('fs').readFileSync;
+const resolve = require('path').resolve;
+
+const parserOpts = {
+  headerPattern: /^(\w*)(?:\((.*)\))?: (.*)$/,
+  headerCorrespondence: [
+    'type',
+    'scope',
+    'message',
+  ],
+};
+
+const writerOpts = {
+  transform: commit => {
+    if (typeof commit.hash === 'string') {
+      commit.hash = commit.hash.substring(0, 7);
+    }
+
+    if (commit.type) {
```

Comment:
should we also de-dupe test and tests while we're in here?

Reply:
fine by me, I just copy pasted it 😊